addons/info/src/components/__snapshots__/PropTable.test.js.snap

```diff
@@ -0,0 +1,20 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PropTable multiLineText should have 2 br tags for 3 lines of text 1`] = `
+Array [
```

Comment:
ESlint rules seem to prevent these from being one line each

Reply:
Eslint shouldn't do anything with js.snap files. Please check your editor settings

Reply:
It's actually this that ESlint doesn't want me to make a 1 liner. https://github.com/storybooks/storybook/pull/2053/files#diff-93c8453ec93a245a8bc84801b42540c7R96. That is causing this snapshot to be so big (not that that really matters)